libminifi/test/libtest/integration/HTTPHandlers.h

```diff
@@ -17,7 +17,6 @@
  */
 #pragma once
 
-// #include <algorithm>
```

Comment:
I'd remove any unused headers completely, not just comment them out.

Reply:
It was removed in https://github.com/apache/nifi-minifi-cpp/pull/2057/commits/4ce50e373a42eba19a68e33a6716bcc17468a60d